Freshli.Web/Models/AnalysisRequest.cs

```diff
@@ -3,11 +3,22 @@
 
 namespace Freshli.Web.Models {
   public class AnalysisRequest {
+    public enum Status {
+      New,
+      InProgress,
+      Success,
+      Invalid,
+      Error,
+      Retrying
+    }
+
```

Comment:
It feels like this should be named `State` (or everything else that is named `State` should be named `Status`) so that there aren't two terms being used.

Reply:
`State` can't be used for both the type name and the property name. I could move it to the outside of the class and then name it `AnalysisRequestState`. Would that be better?

Reply:
Maybe?

Reply:
Renamed in 0259412. How's that?

Reply:
Accidentally left it as `status`. 83dec44 fixes that.

Reply:
Looks good! 🎉